src/main/java/com/suse/saltstack/netapi/config/ClientConfig.java

```diff
@@ -35,6 +35,13 @@
     public static final Key<String> PROXY_PASSWORD = new Key<>();
 
     /**
+     * Maximum websocket message length in characters. The default value corresponds to
+     * 10 MB, assuming that one character takes up 2 bytes. This limit can be disabled
+     * by setting a value less than or equal to 0.
+     */
+    public static final Key<Integer> WEBSOCKET_MAX_MESSAGE_LENGTH = new Key<>(5242880);
```

Comment:
I am still for using hexadecimals here. And the doc above is anyway confusing and seems to be incorrect. If your data is two-bytes, then the actual limit supposed to be doubled. That is, you allow now 5MB only per one byte.